fcrepo-import-export-common/src/main/java/org/fcrepo/importexport/Constants.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.importexport;
+
+/**
+ * @author Mike Durbin
+ */
+public abstract class Constants {
```

Comment:
I'm not... I did it that way first, then reconsidered because the name of the file seemed misleading because the BINARY_EXTENSION constant isn't an "fcrepo" thing it's an "import-export" thing.  That said, it's an "fcrepo"-import-export thing.